dockerfiles/ci/xfail_tests/8.2.list

```diff
@@ -36,6 +36,8 @@ Zend/tests/weakrefs/weakmap_basic_map_behavior.phpt
 Zend/tests/weakrefs/weakmap_error_conditions.phpt
 ext/curl/tests/bug45161.phpt
 ext/curl/tests/bug64267.phpt
+ext/curl/tests/bug68937.phpt
+ext/curl/tests/bug68937_2.phpt
```

Comment:
**I don't know** why these are 'failing' on the CI. I say 'failing' because when I SSH into CircleCI's job and run `run-tests.php` using the same command, it passes as it should; however, these two tests _never end_ when run 'by' CircleCI.